Modules/getpath.py

```diff
@@ -375,6 +375,25 @@ def search_up(prefix, *landmarks, test=isfile):
                     pass
                 if not base_executable:
                     base_executable = joinpath(executable_dir, basename(executable))
+                    # It's possible "python" is executed from within a posix venv but that
+                    # "python" is not available in the "home" directory as the standard
+                    # `make install` does not create it and distros often do not provide it.
+                    #
+                    # In this case, try to fall back to known alternatives
+                    if os_name != 'nt' and not isfile(base_executable):
+                        _base_exe = basename(executable)
```

Comment:
Leading underscores on variables isn't a convention that we use in this file.

Reply:
addressed